accord-core/src/main/java/accord/local/Node.java

```diff
@@ -382,7 +382,7 @@ public void onEpochClosed(Ranges ranges, long epoch)
     @Override
     public void onEpochRetired(Ranges ranges, long epoch)
     {
-        topology.onEpochRedundant(ranges, epoch);
+        topology.onEpochRetired(ranges, epoch);
```

Comment:
we spoke about this in slack.  This should have been renamed a long time ago, and since im touching this logic i am doing that rename now